mint/ImpactTable.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Really this could just be called `DynamicTable` or similar - there's nothing actually particular to the impact data here.